packages/reports/tests/integration/components/report-actions/export-test.js

```diff
@@ -56,11 +64,13 @@ module('Integration | Component | report actions - export', function (hooks) {
     await render(TEMPLATE);
     await click('.report-actions__export-btn');
 
-    assert.equal(
-      find('.export__download-link').getAttribute('download'),
-      'hyrule-news',
-      'The download attribute is set correctly'
-    );
+    assert
+      .dom('.export__download-link')
+      .hasAttribute(
+        'href',
+        /(?=.*?data.naviapp.io)(?=.*format=csv)(?=.*dateTime=)(?=.*csv&filename=Hyrule%20News-\d\d\d\d\d\d\d*T\d\d\d\d\d\d)/,
+        'csv export url is generated for the report'
+      );
```

Comment:
we test the url in the previous test, this is for the `download` attr so maybe it should be something like 
```suggestion
    assert
      .dom('.export__download-link')
      .hasAttribute(
        'download',
        /hyrule-news-\d\d\d\d\d\d\d*T\d\d\d\d\d\d)/,
        'The download attribute is set correctly'
      );
```

Reply:
Before I have used filename attribute, So because of Url Encoding Special Characters getting added. Now i have appended in the url itself instead of passing into the attribute.

In Previous also we have tested by appending into the url. Let me change into appending into the url in` export.ts`

Reply:
let me know if I can go with filename attribute and url appending.